core/src/main/java/com/linecorp/armeria/common/util/TimeoutController.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Initializes the timeout scheduler with the specified {@code timeoutMillis}.
```

Comment:
Initializes -> Sets?
Do you have a reason that you changed the name? I think I'd like `initTimeout`. 🤔

Reply:
I didn’t update Javadoc yet. I’ll let you know when ready to review. 🙇‍♂️